zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java

```diff
@@ -16,7 +16,8 @@ public enum ClusterConfigurationRequestTopics {
   SCALE_MEMBERS("topology-member-scale"),
   QUERY_TOPOLOGY("topology-query"),
   CANCEL_CHANGE("topology-change-cancel"),
-  FORCE_SCALE_DOWN("topology-force-scale-down");
+  FORCE_SCALE_DOWN("topology-force-scale-down"),
+  DISABLE_EXPORTER("topology-exporter-disable");
```

Comment:
:sweat_smile: "topology"? I get the why though :smile:

Reply:
:see_no_evil: I feel it weird to only name this one differently. That said, I would like to rename all of it, but have to do it in a backward compatible way. May be one version that can listens to both topic and the next version which removes the old ones :thinking: